stock-usecase/src/main/java/org/dhis2/android/rtsm/services/rules/RuleValidationHelperImpl.kt

```diff
@@ -240,7 +240,7 @@ class RuleValidationHelperImpl @Inject constructor(
                     event.organisationUnit()!!,
                     d2.organisationUnitModule()
                         .organisationUnits().uid(event.organisationUnit())
-                        .blockingGet()!!.code(),
+                        .blockingGet()?.code() ?: "",
```

Comment:
it's the same doubt that in previous comments: if `data class RuleEvent(...)` has it's property `val organisationUnitCode: String?,` why can't the assignment be null?